CHANGES.rst

```diff
@@ -466,6 +466,8 @@ Bug fixes
   - Included a new command-line script called ``fitsinfo`` to display
     a summary of the HDUs in one or more FITS files. [#3677]
 
+  - The function ``report_diff_values()`` now explicitly reports type differences. [#4122]
```

Comment:
I may end up asking you to move this depending on when this gets merged, but we'll see.